### DISCUSSION TITLE
Sprint 2 Planning

### DISCUSSION THREAD

**Discussion by samwise-311** [cs481-ekh/f22-fires-wild#117] [Meeting Minutes]

**Comment by collins-benjaminw** [comment]
9/27/2022
Excused: 0
Absent:0
New Roles:
  Team Lead - Brenden
  Scribe - Benjamin
  Product Owner - Jeremy
  Devs - David, Samuel

Summary:
Goal: We want to have the minimum viable product by the end of this sprint, detailed in its own discussion

- We are committing to improve our tasking structure by having tasks written before starting code
- We are moving our scrums to thursdays during class time

Action Items:
- Collisions on database need to be addressed with sponsor (email)
- Draft story on UI elements (colors, pre-built css?)
- Everyone should use research on current stories to drive future story writing
